docs/io/fits/appendix/faq.rst

```diff
@@ -355,6 +341,7 @@ If the data must be returned to integers before saving, use the `ImageHDU.scale
     >>> image.scale('int32')
     >>> image.header['BITPIX']
     32
+    >>> hdul.close()
```

Comment:
What's with the ____ before >>> here? Does this indicate there are tabs or something like that?

Reply:
It's a Sphinx link that "normal RST" doesn't recognize a few lines above. It thinks this is all one huge link. It renders correctly in the documentation: (link hasn't changed: http://docs.astropy.org/en/latest/io/fits/appendix/faq.html#id15)

Reply:
should be fixed now.